plugin/parser/differ/pg/test-data/test_differ_data.yaml

```diff
@@ -70,3 +70,23 @@
     ALTER TABLE "s1"."t2"
         DROP COLUMN "b";
     DROP TABLE "s1"."t1", "s2"."t4";
+- oldSchema: |
```

Comment:
Add a case for the same checks with different ordering

```
    create table public.t1(
      a int,
      b int,
      constraint check_a check(a > 1),
      constraint check_b check(b > 1)
    );
  newSchema: |
    create table public.t1(
      a int,
      b int,
      constraint check_b check(b > 1),
      constraint check_a check(a > 1)
    )
```

Reply:
Updated.